.circleci/config.yml

```diff
@@ -23,7 +23,35 @@ commands:
             echo 'nvm alias default v8.11.2' >> $BASH_ENV
 
 jobs:
-
+  build-base:
+    docker:
+      - image: docker:18.06.3-ce-git
+    steps:
+      - setup_remote_docker
```

Comment:
I'm not clear on the difference between the `remote` executor and the `machine` executor. Do you know?

https://circleci.com/docs/2.0/building-docker-images/#example

Reply:
As far as I understood it from the docs, the `machine executor` spins up a dedicated vm build environment, where it's possible to even use volumes, system resources, modify sysctl etc.   
The `docker executor` just loads a container and runs the desired steps in it without the functions of a dedicated system env/resources.
I chose the `docker executor` mainly because it's not necessary to spin up an entire dedicated build env since the steps to build the image are not using any sophisticated features (and don't need to), plus the `docker executor` can be executed in the local runner (`machine` not).